blocks/editable/format-toolbar/index.js

```diff
@@ -130,6 +146,13 @@ class FormatToolbar extends Component {
 				isActive: !! formats[ control.format ],
 			} ) );
 
+		// TODO: make this not look hideous
+		const linkSettings = settingsVisible && (
```

Comment:
Have you considered using the `Popover` component?

Reply:
Nope, mainly because I didn't know what it was :smile: Looks like a good way to have this visibility state managed for us, although I don't think it would help with my awful design skills?

Reply:
I've switched this to use a popover, but it still looks really bad. I'm not sure what markup we need to have this render like the wireframes, and there's nowhere that we're actually using a PopoverContext for me to see how to get this rendering nicely, either. Also, there are focus issues once you toggle the checkbox (click the settings icon, toggle the checkbox, then you have to click the settings icon twice to close the Popover, as I think the first click takes the focus away from the Popover, which feels bad.)